book/listings/scalable_lists/1/integer_object/mod.rs

```diff
@@ -0,0 +1,29 @@
+mod imp;
+
+use glib::Object;
+use gtk::glib;
+use gtk::prelude::*;
+
+// ANCHOR: integer_object
+glib::wrapper! {
+    pub struct IntegerObject(ObjectSubclass<imp::IntegerObject>);
+}
+
+impl IntegerObject {
+    pub fn new() -> Self {
+        Object::new(&[]).expect("Failed to create IntegerObject")
+    }
+
+    pub fn from_integer(number: i32) -> Self {
```

Comment:
```suggestion
    impl From<i32> for IntegerObject { ... }`
```

Reply:
I was contemplating if I should use the `From` trait here.
If you want to use it, then this decides it :)